_ml-commons-plugin/api/model-apis/register-model.md

```diff
@@ -367,6 +367,20 @@ Field | Data type | Description
 
 The input and output fields will be evaluated against the separately provided JSON schema. You do not necessarily need to provide both input and output fields simultaneously.
 
+Specifically, for users' convenience, we have provided a set of predefined model interface that can be generated automatically during model registration, if the user choose to follow the [connector blueprints]({{site.url}}{{site.baseurl}}/ml-commons-plugin/remote-models/blueprints/). 
+The predefined model interface is generated based on the connector blueprint and the model's metadata, so user needs to follow the blueprint to create the connector strictly in order to avoid exceptions.
```

Comment:
```suggestion
The predefined model interface is generated based on the connector blueprint and the model's metadata, so the user must strictly follow the blueprint when creating the connector in order to avoid exceptions.
```